contracts/interfaces/ILockedNORIV2

```diff
@@ -0,0 +1,73 @@
+pragma solidity ^0.8.10;
+
+interface ILockedNORIV2 {
```

Comment:
Looks like a bug that this in missing an file extension.

Reply:
Yeah quickly generated these. So probably just a bug on my cli use